metaflow/plugins/argo/argo_workflows.py

```diff
@@ -205,6 +248,14 @@ def list_templates(flow_name, all=False):
     def delete(name):
         client = ArgoClient(namespace=KUBERNETES_NAMESPACE)
 
+        workflow_template = client.get_workflow_template(name)
+        sensor_name = workflow_template["metadata"]["annotations"].get(
+            "metaflow/sensor_name", name.replace(".", "-")
+        )
+        sensor_namespace = workflow_template["metadata"]["annotations"].get(
```

Comment:
There's a slight problem with using `ARGO_EVENTS_SENSOR_NAMESPACE` as a fallback here. It should probably be `KUBERNETES_NAMESPACE` instead as:
- if the sensor_namespace annotation is missing, the flow was deployed before the custom sensor namespace feature (so it will only exist in the default kubernetes namespace)
- if `ARGO_EVENTS_SENSOR_NAMESPACE` is set to a custom one, and the flow was deployed before this feature, then `argo-workflows delete` will fail to delete the sensor.

Reply:
Good catch.  Changing it and adding a code comment.